src/makeRobot.js

```diff
@@ -38,7 +38,74 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
```suggestion
  return {
```
You can return this object right away